### PR TITLE
Fixed broken tests for io package

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

Fixed broken tests for io package

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tenable/io/tags.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def create(self, category, value, description=None, category_description=None,
 
             >>> tio.tags.create('00000000-0000-0000-0000-000000000000', 'Madison')
         '''
-        all_permissions = ['ALL', 'CAN_EDIT', 'CAN_SET_PERMISSIONS', 'CAN_USE']
+        all_permissions = ['ALL', 'CAN_EDIT', 'CAN_SET_PERMISSIONS']
```
Comment (adabala-durga) [inline tenable/io/tags.py:227]: Don't we need CAN_USE?

File: tenable/io/access_groups.py [hunk 2]
```diff
@@ -214,9 +214,12 @@ def edit(self, id, **kw):
             'all_users': self._check('all_users', g['all_users'], bool),
             'all_assets': self._check('all_assets', g['all_assets'], bool),
             'rules': g['rules'],
-            'principals': g['principals']
+            # 'principals': g['principals']
```
Comment (adabala-durga) [inline tenable/io/access_groups.py:217]: Can we remove commented code?

File: tests/io/conftest.py [hunk 3]
```diff
@@ -65,7 +65,7 @@ def teardown():
 @pytest.fixture
 def user(request, api):
     user = api.users.create(
-        '{}@pytenable.io'.format(uuid.uuid4()),
+        '{}@tenable.com'.format(uuid.uuid4()),
```
Comment (adabala-durga) [inline tests/io/conftest.py:68]: Do we need this change or added accidentally?

File: tenable/io/access_groups.py (hunk 2 above)
Comment (tushar-balwani) [reply tenable/io/access_groups.py]: removed commented code

File: tenable/io/tags.py (hunk 1 above)
Comment (tushar-balwani) [reply tenable/io/tags.py]: reverted changes in this PR and created PR #340 for bugfix

**Review by mvalliammal — CHANGES_REQUESTED** [review]
63:16: Unused variable 'err' (unused-variable)
209:8: Variable name "g" doesn't conform to snake_case naming style

**Review by mvalliammal — APPROVED** [review]
I approved the changes. Looks good

**Pull request merged**